students/kulon_aleksander/lesson_04/unit_tests_distance.py

```diff
@@ -0,0 +1,35 @@
+import math
```

Comment:
Don't mix ```unitest``` and ```pytest``` modules. Rewrite this by using only the pytest module. Remember about the proper naming convention.